fs/params/camp_params.csv

```diff
@@ -19,4 +19,14 @@ Haman-al-Alil,30-39,0.42,1.85,30,9.49,,,,
 Haman-al-Alil,40-49,0.51,5.43,30,7.65,,,,
 Haman-al-Alil,50-59,0.59,15.05,30,4.32,,,,
 Haman-al-Alil,60-69,0.72,33.29,30,1.81,,,,
-Haman-al-Alil,70+,0.76,61.76,30,0.9,,,,
\ No newline at end of file
+Haman-al-Alil,70+,0.76,61.76,30,0.9,,,,
+
+
+Mumbai,0-9,0.4,0.76,30,36.12,9465,,,
+Mumbai,10-19,0.25,0.81,30,27.81,,,,
+Mumbai,20-29,0.37,0.99,30,11.9,,,,
+Mumbai,30-39,0.42,1.85,30,9.49,,,,
+Mumbai,40-49,0.51,5.43,30,7.65,,,,
+Mumbai,50-59,0.59,15.05,30,4.32,,,,
+Mumbai,60-69,0.72,33.29,30,1.81,,,,
+Mumbai,70+,0.76,61.76,30,0.9,,,,
```

Comment:
are these manually added? where do they come from?

Reply:
yes i manually added this for testing purpose

Reply:
ok, please remove them before merging